app/models/setting/content.rb

```diff
@@ -150,7 +150,9 @@ def self.default_settings
       self.set('autosearch_while_typing', N_('For pages that support it, automatically perform search while typing in search input.'),
                true, N_('Autosearch')),
       self.set('autosearch_delay', N_('If Autosearch is enabled, delay in milliseconds before executing searches while typing.'),
-               500, N_('Autosearch delay'))
+               500, N_('Autosearch delay')),
+      self.set('bulk_load_size', N_('The amount of items fetched from a single paged Pulp API call.'), 100,
```

Comment:
```suggestion
      self.set('bulk_load_size', N_('The number of items fetched from a single paged Pulp API call.'), 100,
```
what are the units?

Reply:
I'm not sure what to call the units besides items from the API response.

Reply:
Oh, I got it now :)  Yeah I think "number of items" is clear enough then.